bittensor/_dendrite/dendrite_impl.py

```diff
@@ -1024,4 +1023,4 @@ def to_wandb( self ):
             return wandb_info
         except Exception as e:
             bittensor.logging.error( prefix='failed dendrite.to_wandb()', sufix = str(e))
-            return {}
```

Comment:
Unrelated but I thought we removed wandb support? Is this function vestigial?

Reply:
yeah just lingering